examples/BUILD

```diff
@@ -15,6 +15,19 @@ py_test(
     ]
 )
 
+py_test(
```

Comment:
is it used by the CI? or just by yourself on your local machine?

Reply:
Just locally, but it's too useful to do without right now!